labelbox_dev/task.py

```diff
@@ -221,3 +223,77 @@ def errors(self) -> Union[Dict[str, Any], None]:
             errors['failed_data_rows'] = task_result['failedDataRows']
 
         return errors if errors else None
+
+
+class UpdateDataRowsTask(BulkTask):
+    """ Asynchronous task representing bulk data_rows update job
+    """
+
+    task_type = TaskType.UPDATE_DATA_ROWS
+
+    def __init__(self, json):
+        super().__init__(json)
+
+    @lru_cache()
+    def fetch_result(self) -> Dict[str, Any]:
```

Comment:
Very good point. We should definitely be encapsulating the task results as typed dicts

Reply:
Ah I remember why I didn't do this. I wanted to return DataRow instances for `UpdateDataRowsTask.results()` and `CreateDataRowsTask.results()` but in order to do so, I need to import DataRow class, which causes circular dependency. Do you happen to have any suggestions on how best to handle these result types?

Reply:
One way to do this would be to convert staticmethods to module level functions, which is a more common pattern in Python compared to staticmethods anyway.

Reply:
I don't think that would help here, unless I'm not understanding the module level pattern correctly. We would still have to import `CreateDataRowsTask` from data_row.py and import `DataRow` from task.py, which would result in circular imports

Reply:
Sorry, I misunderstood the problem, maybe we should move the `UpdateDataRowsTask` into `data_row.py`? Another way could be to pass in the `DataRow` class into the `CreateDataRowsTask` constructor instead of importing it?